test/test-parse.cpp

```diff
@@ -539,6 +539,80 @@ TEST(test_escape_first_character_in_keyword,
   EXPECT_EQ(escape_first_character_in_keyword(u8"b1n z"_sv), u8"\\u{62}1n z");
   EXPECT_EQ(escape_first_character_in_keyword(u8"ZYXW"_sv), u8"\\u{5a}YXW");
 }
+
+#if QLJS_HAVE_SETJMP
+TEST(test_no_overflow, parser_depth_limit_not_exceeded) {
+  {
+    string8 opening_parens(300, '(');
```

Comment:
Instead of repeating `300` everywhere, we should make a constant. Perhaps make `parser::limit_` public and reuse it?

Reply:
Done